Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.7.0"
 
 [deps]
 DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
+Dierckx = "39dd38d3-220a-591b-8e3c-4c3a8c710a94"
```

Comment:
Note that you should also add compatibility bounds for the package

Reply:
TIL about compatibility bounds. It looks like versions 0.4.1 and 0.4.2 support Julia 1.0, but I was wondering if there was a hard and fast rule for how far back to go?

Reply:
> TIL about compatibility bounds.

A `Project.toml` with compat bounds and a `Manifests.toml` will become your best friends when you want to prepare a script for your analysis pipeline.  You can read more in https://julialang.github.io/Pkg.jl/v1/environments/.

> It looks like versions 0.4.1 and 0.4.2 support Julia 1.0, but I was wondering if there was a hard and fast rule for how far back to go?

You can simply use
```toml
Dierckx = "0.4"
```
see https://julialang.github.io/Pkg.jl/v1/compatibility/

Reply:
Ok, should be up now! cbd9d404e3ec9be3762969c82660be578a2d94e0

Thanks for the additional info. It's definitely been a shift coming from conda land.

Reply:
> It's definitely been a shift coming from conda land.

The idea is not much different, you'll create an environment also in this case.  It's just different how you do that.  And the environment is local to Julia, it doesn't overtake your shell, which caused me troubles a few times when trying to run unrelated system programs.